tools/handle-ddl-lock.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Handling the Sharding DDL Lock Manually
+summary: Learn to manually unlock the sharding DDL lock.
+category: tools
+---
+
+# Handling the Sharding DDL Lock Manually
+
+This document shows how to manually unlock the sharding DDL lock in different cases.
+
+Generally, the sharding DDL lock synchronization of DM can be completed automatically. But when some abnormality happens, you need to use `unlock-ddl-lock`/`break-ddl-lock` to unlock the abnormal DDL lock manually.
+
+The possible causes of an abnormality include:
+
+- Some DM-workers go offline
+- Some DM-worker restarts (or is unreachable temporarily)
```

Comment:
"Some DM-worker" is mixing singular/plural.  If the condition is true for single, we should remove plural for readability:
- A DM-worker goes offline
- A DM-worker restarts (or is temporarily unreachable)